stylesheets/lib/postcss-rule-per-line.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+module.exports = (opts) => {
+  return {
+    postcssPlugin: 'postcss-rule-per-line',
+    OnceExit (root) {
+      root.walk((node) => {
+        if (node.type.endsWith('rule') && node.prev()) node.raws.before = '\n'
+      })
+      root.raws.after = '\n'
+    },
```

Comment:
Ditto.
I guess the goal is to add a newline before and after each rule (otherwise the minified CSS is a single line) right? But is that temporary? Or should we remove it at a later stage?

Reply:
I find single-line minified files to be impossible to inspect. I've always broken the rules out on each line in the default stylesheet, and it has proven to be a good strategy. So I'm sticking with it here.